resources/public/api/conf/2.0/application.raml

```diff
@@ -98,6 +98,19 @@ documentation:
             description: 'The tax year the data applies to, for example, 2021-22. The start year and end year must not span two tax years. The minimum tax year is 2017-18 and the maximum tax year is 2021-22. No gaps are allowed, for example, 2020-22 is not valid.'
             example: 2021-22
         get: !include uk_property_non_fhl_retrieve.raml
+  /furnished-holiday-lettings:
```

Comment:
The url should be /individuals/business/property/uk/furnished-holiday-lettings/{nino}/{taxYear}
uk is missing from it